### ISSUE TITLE
Adding Gradio Support for UI 

### ISSUE THREAD

**Issue by neokd** [neokd/NeoGPT#103]
I am seeking to integrate Gradio into our project to create a user-friendly interface for running machine learning models. Gradio simplifies UI development and can be easily incorporated into the project.

**Comment by savyez** [comment]
what are we seeking here? I mean are we seeking to integrate the gradio chat Interface?

**Comment by neokd** [comment]
@savyez yes as an alternative to the streamlit one.

**Comment by savyez** [comment]
@neokd is there any specific advantage of integrating gradio to this model? streamlit support seems ok to me. 🤔

**Comment by neokd** [comment]
@savyez Streamlit looks very good but just had a thought of integrating another UI like gradio or chainlit something like that.

**Comment by savyez** [comment]
@neokd  I would suggest we wait till issue #104 is solved. after that adding gradio to the updated ui.py would be a good thing. what are your thoughts?

**Comment by neokd** [comment]
Yeah. I thought about gradio. Streamlit looks fine and have good support with a decent UI. So for now we can stick to streamlit and then add support for gradio too

**Comment by neokd** [comment]
Closing this issue since i think we can develop better with streamlit itself
